src/screens/MyPromotions.tsx

```diff
@@ -1,108 +1,24 @@
 import '../index.less';
+import './MyPromotions.less';
 
-import { Checkbox, Col, Row } from 'antd';
-import React, { CSSProperties, ReactElement, useCallback, useEffect, useState } from 'react';
+import { Button, Col, Row } from 'antd';
+import React, { ReactElement, useCallback, useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
 
-import UploadPromoButton from '../components/button/UploadPromoButton';
-import DropdownMenu from '../components/DropdownMenu';
 import DeleteModal from '../components/modal/DeleteModal';
 import PromotionCard from '../components/promotion/PromotionCard';
 import { useAuthUser } from '../contexts/AuthUserContext';
 import PromotionService from '../services/PromotionService';
 import UserService from '../services/UserService';
-import { Dropdown, DropdownType } from '../types/dropdown';
 import { Promotion, VoteState } from '../types/promotion';
 
-const dropdowns: Dropdown[] = [
-  {
-    text: 'Sort',
-    type: DropdownType.Radio,
-    options: [
-      {
-        action: () => {
-          /* stub */
-        },
-        description: 'Sort by Option 1',
-        text: 'Option 1',
-      },
-      {
-        action: () => {
-          /* stub */
-        },
-        description: 'Sort by Option 2',
-        text: 'Option 2',
-      },
-      {
-        action: () => {
-          /* stub */
-        },
-        description: 'Sort by Option 3',
-        text: 'Option 3',
-      },
-    ],
-  },
-  {
-    text: 'Category',
-    type: DropdownType.MultiSelect,
-    options: [
-      {
-        action: () => {
-          /* stub */
-        },
-        text: 'Option 1',
-      },
-      {
-        action: () => {
-          /* stub */
-        },
-        text: 'Option 2',
-      },
-      {
-        action: () => {
-          /* stub */
-        },
-        text: 'Option 3',
-      },
-    ],
-  },
-];
-
-const dropdownMenuWidth = 30;
-const styles: { [identifier: string]: CSSProperties } = {
-  body: {
-    alignItems: 'center',
-    justifyContent: 'space-between',
-    padding: 20,
-    paddingTop: 20,
-    width: '100%',
-  },
-  dropdownMenuContainer: {
-    width: `${dropdownMenuWidth}%`,
-  },
-  checkBoxContainer: {
-    padding: 20,
-    width: `${100 - dropdownMenuWidth}%`,
-  },
-  promotions: {
-    marginTop: 15,
-  },
-  uploadPromoButtonContainer: {
-    position: 'fixed',
-    bottom: 50,
-    right: 50,
-  },
-};
-
-const onChange = (): void => {
-  /* stub */
-};
-
 export default function MyPromotions(): ReactElement {
   const [isModalVisible, setIsModalVisible] = useState<boolean>(false);
```

Comment:
Hmm, this is because the BE doesn't tell you vote state nor isSaved by for the uploadedPromotions endpoint... i don't think I can get to it on time ):


![image](https://user-images.githubusercontent.com/49849754/114284395-66318480-9a04-11eb-9a09-f43bf7c3ddfa.png)

Reply:
Oh I caught this too no problem -- I asked Yu to take a look at that and he's working on it!

Reply:
Wait actuallly I can fix this, making a PR for it rn

Reply:
Ohcrap

Reply:
I already started on it

Reply:
BE PR https://github.com/ubclaunchpad/PromoPal-backend/pull/138